### ISSUE TITLE
Syntax error on generated Update statement. 

### ISSUE THREAD

**Issue by eddiellopez** [agrosner/DBFlow#1598]
DBFlow Version: 4.2.4

Bug or Feature Request: Review the syntax error and/or specify if the feature being used is an invalid or unsupported use case. Fix if it is a bug.

Description:
When trying to use a foreign model key as the primary key of another model, and the first have a composed primary key, the Model_Table generated update statement has a syntax error.

Consider the following class definition: 
```Java
@Table(database = People.class)
public class Person extends BaseModel {
    @PrimaryKey
    private String name;

    @PrimaryKey
    private String lastName;

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getLastName() {
        return lastName;
    }

    public void setLastName(String lastName) {
        this.lastName = lastName;
    }
}
```

Referenced as foreign key by the following entity:
```Java
@Table(database = People.class)
public class Pet extends BaseModel {

    @ForeignKey(stubbedRelationship = true)
    @PrimaryKey
    private Person person;

    @PrimaryKey
    private String name;

    public Person getPerson() {
        return person;
    }

    public void setPerson(Person person) {
        this.person = person;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }
}
```
The generated update state looks like the folowing:
```Java
  @Override
  public final String getUpdateStatementQuery() {
    return "UPDATE `Pet` SET `person_name`=?,`person_lastName`=?,`name`=? WHERE `person_name`=?,`person_lastName`=? AND `name`=?";
  }
```

Which may have an `AND` keyword missing and fails in runtime:
```
    Process: com.eddiellopez.dbflowtest, PID: 11025
    java.lang.RuntimeException: Unable to start activity ComponentInfo{com.eddiellopez.dbflowtest/com.eddiellopez.dbflowtest.MainActivity}: android.database.sqlite.SQLiteException: near ",": syntax error (code 1): , while compiling: UPDATE `Pet` SET `person_name`=?,`person_lastName`=?,`name`=? WHERE `person_name`=?,`person_lastName`=? AND `name`=?
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2778)
        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2856)
        at android.app.ActivityThread.-wrap11(Unknown Source:0)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1589)
        at android.os.Handler.dispatchMessage(Handler.java:106)
        at android.os.Looper.loop(Looper.java:164)
        at android.app.ActivityThread.main(ActivityThread.java:6494)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:438)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:807)
     Caused by: android.database.sqlite.SQLiteException: near ",": syntax error (code 1): , while compiling: UPDATE `Pet` SET `person_name`=?,`person_lastName`=?,`name`=? WHERE `person_name`=?,`person_lastName`=? AND `name`=?
        at android.database.sqlite.SQLiteConnection.nativePrepareStatement(Native Method)
        at android.database.sqlite.SQLiteConnection.acquirePreparedStatement(SQLiteConnection.java:890)
        at android.database.sqlite.SQLiteConnection.prepare(SQLiteConnection.java:501)
        at android.database.sqlite.SQLiteSession.prepare(SQLiteSession.java:588)
        at android.database.sqlite.SQLiteProgram.<init>(SQLiteProgram.java:58)
        at android.database.sqlite.SQLiteStatement.<init>(SQLiteStatement.java:31)
        at android.database.sqlite.SQLiteDatabase.compileStatement(SQLiteDatabase.java:1070)
        at com.raizlabs.android.dbflow.structure.database.AndroidDatabase.compileStatement(AndroidDatabase.java:56)
        at com.raizlabs.android.dbflow.structure.ModelAdapter.getUpdateStatement(ModelAdapter.java:130)
        at com.raizlabs.android.dbflow.structure.ModelAdapter.getUpdateStatement(ModelAdapter.java:71)
        at com.raizlabs.android.dbflow.sql.saveable.ModelSaver.save(ModelSaver.java:30)
        at com.raizlabs.android.dbflow.structure.ModelAdapter.save(ModelAdapter.java:186)
        at com.raizlabs.android.dbflow.structure.BaseModel.save(BaseModel.java:62)
        at com.eddiellopez.dbflowtest.MainActivity.onCreate(MainActivity.java:29)
        at android.app.Activity.performCreate(Activity.java:7009)
        at android.app.Activity.performCreate(Activity.java:7000)
        at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1214)
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2731)
        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2856) 
        at android.app.ActivityThread.-wrap11(Unknown Source:0) 
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1589) 
        at android.os.Handler.dispatchMessage(Handler.java:106) 
        at android.os.Looper.loop(Looper.java:164) 
        at android.app.ActivityThread.main(ActivityThread.java:6494) 
        at java.lang.reflect.Method.invoke(Native Method) 
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:438) 
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:807) 

```

Thanks!

**Comment by masterhaowu** [comment]
I am having the same issue.

**Comment by agrosner** [comment]
fixed in 5.0.0-alpha1
